packages/react-charting/src/components/DonutChart/DonutChartRTL.test.tsx

```diff
@@ -0,0 +1,108 @@
+import { render, screen, queryAllByAttribute, fireEvent } from '@testing-library/react';
+import { chartPoints } from './DonutChart.test';
+import { DonutChart } from './index';
+import * as React from 'react';
+
+test('Should hide callout on mouse leave', () => {
+  // Arrange
+  const { container } = render(<DonutChart data={chartPoints} innerRadius={55} calloutProps={{ doNotLayer: true }} />);
+
+  // Act
+  const getById = queryAllByAttribute.bind(null, 'id');
+  fireEvent.mouseOver(getById(container, /Pie/i)[0]);
+  expect(getById(container, /callout/i)[0]).toBeDefined();
```

Comment:
can we also match the snapshot in this case

Reply:
done